src/integration/car-connector/CarConnectorFactory.ts

```diff
@@ -8,13 +9,14 @@ import TenantComponents from '../../types/TenantComponents';
 import Utils from '../../utils/Utils';
 
 export default class CarConnectorFactory {
-  static async getCarConnectorImpl(tenant: Tenant, connectorId: string): Promise<CarConnectorIntegration<CarConnectorSetting>> {
+  static async getCarConnectorImpl(tenant: Tenant, connectorId: string, car?: Car): Promise<CarConnectorIntegration<CarConnectorSetting>> {
```

Comment:
The second param is more a connectionID I think.
I wouldn't expect the Car param here (maybe due to Mercedes.)

Reply:
Done. 
You are right. We will only need the connectorId in the future, it will be saved in the car object and we can directly pass this property to the method.